### PR TITLE
English documents update

### PR DESCRIPTION
**Pull request by FangyangJz**
fix and add some contents in docs/zh-cn/

本次 PR 内容，
docs/en-us 添加相关文件, 完成_sidebar.md, advanced.md翻译工作, 其余文件有待更新; 
修改docs/zh-cn下donors和donate文件, 以兼容英文版本;
修正中文手册中部分错误, docs/zh-cn/advanced.md

### CODE REVIEW

**Comment by chenjiandongx** [comment]
@FangyangJz 这 PR 你慢慢维护着。差不多再来合并，你可以持续推送 commit 上来

**Comment by chenjiandongx** [comment]
@FangyangJz 统一为 pyecharts 不要 Pyecharts

**Comment by FangyangJz** [comment]
好的, 各位如果有什么问题可以及时反馈给我

File: docs/en-us/release-note/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## 开发里程碑
```
Comment (chfw) [inline docs/en-us/release-note/README.md]: Still Chinese here

File: docs/en-us/_sidebar.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
+Pyecharts is a class library for generating Echarts charts. Echarts is a data visualization JS library of Baidu Open Source. The graphics generated with Echarts are very visual, and pyecharts is designed to interface with Python, making it easy to use data generation diagrams directly in Python.
```
Comment (chfw) [inline docs/en-us/_sidebar.md]: please reference the description here:https://github.com/vinta/awesome-python/pull/1107

File: docs/en-us/release-note/README.md (hunk 1 above)
Comment (FangyangJz) [reply docs/en-us/release-note/README.md]: ok

**Comment by chfw** [comment]
I suggest that we merge this or first.improvements can be submitted in a new or.

**Comment by FangyangJz** [comment]
I just test the effect of clicking 'Update branch' button.
Some new updates in my local, but meet some problem when git push to my remote repository.
Learning how to solve it ...

**Comment by FangyangJz** [comment]
haha, problem solved. 
Chinese document translation work have completed almost 70%. 
The rest of files are charts_base, charts_custom. And some proofreading work still need to do.
I saw there is a new charts_style file. Maybe it would synchronize after all work finished on this step.

**Comment by chfw** [comment]
Great. Will do the proof reading later.

File: docs/zh-cn/themes.md [hunk 3]
```diff
@@ -124,7 +124,7 @@ bar = Bar()
 
 ## 使用自己构建的主题
 
-Echarts 提供了[主题构建工具](http://echarts.baidu.com/theme-builder/)，你可以从中构建喜欢的主题，如 `myTheme.js`。然后 hack *echarts-themes-pypkg* 包。具体操作如下
```
[the remark this answers is not in the document]
Comment (FangyangJz) [reply docs/zh-cn/themes.md]: 好的, 下次改回来, 理解万岁,  ^_^

File: docs/en-us/charts_configure.md [hunk 4]
```diff
@@ -0,0 +1,853 @@
...
+## General options
+
+> All general configurations are set in```add()```
+
+### xyAxis
+**x, y axis in cartesian coordinate system (Line, Bar, Scatter, EffectScatter, Kline)**
+
+* is_convert -> bool  
+
+    It specifies whether to convert xAxis and yAxis.
```
Comment (chenjiandongx) [inline docs/en-us/charts_configure.md]: 你在 bool 后面加**两个空格**就可以了哈，不用新增空行

File: docs/en-us/charts_base.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
+<<<<<<< HEAD
```
Comment (chenjiandongx) [inline docs/en-us/charts_base.md]: 冲突留下来的“尾巴“

File: docs/en-us/charts_configure.md (hunk 4 above)
Comment (FangyangJz) [reply docs/en-us/charts_configure.md]: 空行是因为参考了目前网页版的效果, 有些英文都团在一起, 感觉有点头晕
空1行在markdown软件中看效果舒服点, 不知道merge后网页效果怎么样.
到时候看看效果吧, 如果不好, 再改下

File: docs/zh-cn/charts_base.md [hunk 6]
```diff
@@ -612,7 +612,7 @@ funnel.add(
 )
 funnel.render()
 ```
-![funnel-demo](https://user-images.githubusercontent.com/19553554/35090186-d8f50db6-fc73-11e7-9b7e-947580a621de.png)
```
Comment (chenjiandongx) [inline docs/zh-cn/charts_base.md]: 这里为什么要改？

Comment (FangyangJz) [reply docs/zh-cn/charts_base.md]: 抱歉, 这个地方搞错了, 替换方向反了

File: docs/zh-cn/charts_base.md [hunk 7]
```diff
@@ -612,7 +612,7 @@ funnel.add(
 )
 funnel.render()
 ```
-![funnel-1](https://raw.githubusercontent.com/pyecharts/pyecharts/master/images/funnel-1.png)
+![funnel-1](https://user-images.githubusercontent.com/19553554/35090186-d8f50db6-fc73-11e7-9b7e-947580a621de.png)
```
Comment (chenjiandongx) [inline docs/zh-cn/charts_base.md]: funnel-demo 统一改为这个吧

**Comment by chfw** [comment]
Thumb up

**Pull request merged**